### ISSUE TITLE
Error: Expected 52 fields, got 47

### ISSUE THREAD

**Issue by pftbest** [borntyping/rust-psutil#19]
Is there any way to work around this error?

Error { repr: Custom(Custom { kind: InvalidInput, error: StringError("Expected 52 fields, got 47 (from /proc/1/stat)") }) }

```
$ uname -a
[16:43:33.4497] Linux (none) 3.3.8-3.9 #4 SMP Thu Jan 18 12:37:58 UTC 2018 mips GNU/Linux
```

**Comment by borntyping** [comment]
Not with the code in it's current state, no. Unfortunately I've never had the time or resources to work out what the parser needs to do on other versions of Linux with slightly different output. I'll be happy to accept pull requests that attempt to fix this though!

**Comment by cjbassi** [comment]
Should be fixed in d5058b7e278c69cac387cf3ac6386a7b1cf55972
